osidb/api_views.py

```diff
@@ -353,6 +353,14 @@ class AffectView(ModelViewSet):
     http_method_names = get_valid_http_methods(ModelViewSet)
     permission_classes = [IsAuthenticatedOrReadOnly]
 
+    def perform_destroy(self, instance):
+        """
+        override the default behavior to proxy the delete to Bugzilla
+        """
+        flaw = instance.flaw
+        instance.delete()
+        flaw.save(bz_api_key=self.request.data.get("bz_api_key"))
```

Comment:
you should use a write-only serializer field instead of grabbing from the request data alone

Reply:
Good point. That would be much cleaner approach. Will do.